x-pack/plugins/observability_solution/inventory/public/components/entities_grid/index.tsx

```diff
@@ -68,12 +72,15 @@ export function EntitiesGrid({
     [entities]
   );
 
+  // For now this is the only option so once we add more we need to change this check as well
```

Comment:
I don't think this comment is needed. I also don't know what `more` means here. If we don't have an issue to link on this comment I'd suggest to remove it.

Reply:
I couldn't find an issue for the other actions and that's out of the scope of the epic but you are right that we don't need a comment, that's obvious that we need to change that when we add more options so I removed it.